src/dpsvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/DpsValidationserviceApplicationTests.java

```diff
@@ -1,14 +1,27 @@
 package ca.bc.gov.open.pssg.rsbc.dps.dpsvalidationservice;
 
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * @author nancymz
  */
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 class DpsValidationserviceApplicationTests {
 
-   /* TO DO - Add test cases */
```

Comment:
TODO - Please change to TODO so IDE finds it.

Reply:
Added the TODO back